google/auth/compute_engine/_metadata.py

```diff
@@ -119,6 +116,11 @@ def ping(request, timeout=_METADATA_DEFAULT_TIMEOUT, retry_count=3):
     #       could lead to false negatives in the event that we are on GCE, but
     #       the metadata resolution was particularly slow. The latter case is
     #       "unlikely".
+    try:
+        timeout = float(os.getenv("GCE_METADATA_TIMEOUT", timeout))
+    except ValueError:  # pragma: NO COVER
+        pass
```

Comment:
I would not capture a ValueError here. Additionally, the timeout variable would not be usable, so the downstream call would have problms.

Reply:
FWIW the original code caught the ValueError. The value is set by the user.
I am thinking if the Exception happens timeout will retain the passed in/default value

Reply:
In that case, can you remove the pragma, and include test coverage for this change?